### ISSUE TITLE
Is it possible to disable disguise on effect?

### ISSUE THREAD

**Issue by bobhenl** [libraryaddict/LibsDisguises#791]
Hello, is it possible to disable disguise in case of player equips effect or so? i have report that players can equip invis potion & disguise as silverfish and then they have so small hitboxes. And I wanted to keep it enabled in PvP but with this I can't.

**Comment by libraryaddict** [comment]
No, but a workaround is you could do something like "/savedisguise silverfish silverfish setinvisible false"
Which means when they disguise as silverfish, they're actually disguising as a silverfish without invisibility.

Though you would need to ensure that they cannot undo this with /modifydisguise, and I'm not 100% but I think you'd need to turn off the setting in config that controls if players can set invisibility on disguises, just to get the initial disguising to work.
